app/models/entities/account_contact.rb

```diff
@@ -19,7 +19,8 @@ class AccountContact < ActiveRecord::Base
   belongs_to :account
   belongs_to :contact
 
-  has_paper_trail class_name: 'Version', meta: { related: :contact }, ignore: [ :id, :created_at, :updated_at, :contact_id ]
+  has_paper_trail class_name: 'Version', meta: { related: :contact },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.